src/routes/docs/products/functions/functions/+page.markdoc

```diff
@@ -141,6 +141,9 @@ and granting access to select [permission roles](/docs/advanced/platform/permiss
 If this is left empty, no user can execute your function.
 Server SDKs, scheduled executions, and event function triggers don't require permissions to execute a function.
 
+You can also control the function scopes to grant it access to project resources such as documents and files.
+If scopes are selected, you can use [dynamic api keys](/docs/products/functions/develop#using-dynamic-api-key) for your function.
+
```

Comment:
This doesn't fit in the "Execute access" section because it is unrelated. It also doesn't match the structure of the rest of this page. Where/how on this page should you mention function scopes? Please also link to documentation on scopes.